todoapp/todos/models.py

```
@@ -1,18 +1,22 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.utils.encoding import smart_text as smart_unicode
-from django.utils.translation import ugettext_lazy as _
-
+from django.utils.encoding import smart_str as smart_unicode
```

Ruff (0.12.2)

`django.utils.encoding.smart_str` imported but unused

Remove unused import: `django.utils.encoding.smart_str`

(F401)